appservice/src/tree/ISiteTreeRoot.ts

```diff
@@ -0,0 +1,11 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ISubscriptionRoot } from "vscode-azureextensionui";
+import { SiteClient } from "../SiteClient";
+
+export interface ISiteTreeRoot extends ISubscriptionRoot {
```

Comment:
What exactly is a site?

Reply:
If it's website, why not IWebsiteTreeRoot?

Reply:
'site' is the terminology used by the arm package. Just being consistent. Same as 'SiteClient'

Reply:
A site is a web app or a function app.

Reply:
I believe I'm supposed to resolve conversations...

Reply:
Lol you're welcome to 'unresolve' before you approve, but as the author of the PR I like to see a list of "Resolved" to make sure I've addressed everything